src/main/java/lesson44/repository/StudentsRepository.java

```diff
@@ -0,0 +1,74 @@
+package lesson44.repository;
+
+import lesson44.model.StudentModel;
+import lesson44.pack.PostgresDriverManager;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class StudentsRepository {
+    @Getter
+    private Map<Integer, StudentModel> integerPersonModelMap;
+    @Autowired
+    public PostgresDriverManager postgresDriverManager;
+    private int COUNTER = 0;
```

Comment:
Why do you need it?

Reply:
PostgresDriverManager is used to connect to the DB.
COUNTER is used as a key for "Map"

Reply:
If you use DB, do you stil need Map?